api/admin/repo.go

```diff
@@ -7,11 +7,10 @@ import (
 	"fmt"
 	"net/http"
 
+	"github.com/go-vela/server/api/types"
```

Comment:
as we move everything to this pattern, any thoughts on giving _every_ types import the `api` alias?

Reply:
I was operating on the following pattern:
- `go-vela/server/api` —> types
- `anything else` —> api

But I'm open to using `api` always if people would prefer that

Reply:
cool that works for me, not blocking either way